planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py

```diff
@@ -120,6 +120,18 @@ def generate_launch_description():
     with open(behavior_path_planner_param_path, "r") as f:
         behavior_path_planner_param = yaml.safe_load(f)["/**"]["ros__parameters"]
 
+    stop_line_param_path = os.path.join(
```

Comment:
You do not need to add this series of change.
Refer to https://github.com/tier4/autoware_launch/pull/271/files#r845688430